Error: Cannot find module 'blanket'

Just did a clean global install of Hercule. First attempt to run it:

```
module.js:338
    throw err;
          ^
Error: Cannot find module 'blanket'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/hercule/lib/main.js:5:13)
    at Object.<anonymous> (/usr/local/lib/node_modules/hercule/lib/main.js:80:4)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
```

Mac OS X Yosemite

It did give me these errors upon installation though:

```
$ sudo npm install -g hercule
/usr/local/bin/hercule -> /usr/local/lib/node_modules/hercule/bin/transclude
npm WARN unmet dependency /usr/local/lib/node_modules/bower/node_modules/tar-fs/node_modules/tar-stream/node_modules/bl requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bower/node_modules/tar-fs/node_modules/tar-stream/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
hercule@0.2.0 /usr/local/lib/node_modules/hercule
├── async@0.9.0
├── pegjs@0.8.0
├── lodash@3.5.0
└── dashdash@1.7.3 (assert-plus@0.1.5)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,16 @@
 language: node_js
 node_js:
-  - "0.10"
+  - '0.10'
+script: npm run travis
 before_script:
   - npm run compile
 after_success:
   - npm run coveralls
+deploy:
+  provider: npm
+  email: git@jwr.vc
+  api_key:
+    secure: Hbd6eYtt4GiBLwru4GetULUTn49pudkDzo3Eof0/FkR7id8eONuPxbl4Wsf/ZB6C0VWBqbY2od/tCCeiUEa5sbEaZT79NQQCnygtZVjC1N/Aynt0Tzq3jjaBP7wOiq5rDzQPVESCloYD6cc0+O+siVz88EzwLzicl0W1tPAwM6o=
+  on:
+    tags: true
+    repo: jamesramsay/hercule

--- a/package.json
+++ b/package.json
@@ -1,12 +1,12 @@
 {
   "name": "hercule",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "description": "Markdown transcluder",
   "main": "./lib/main",
   "scripts": {
-    "test": "./node_modules/.bin/mocha test --compilers coffee:coffee-script/register --reporter spec",
-    "coverage": "./node_modules/.bin/mocha --compilers coffee:coffee-script/register --require blanket --reporter html-cov > coverage.html && open coverage.html",
-    "coveralls": "./node_modules/.bin/mocha --compilers coffee:coffee-script/register --require blanket --reporter mocha-lcov-reporter | ./node_modules/coveralls/bin/coveralls.js",
+    "test": "NODE_ENV=test ./node_modules/.bin/mocha test --compilers coffee:coffee-script/register --reporter spec",
+    "coverage": "NODE_ENV=test COVERAGE=1 ./node_modules/.bin/mocha --compilers coffee:coffee-script/register --require blanket --reporter html-cov > coverage.html && open coverage.html",
+    "travis": "npm run test && NODE_ENV=test COVERAGE=1 ./node_modules/.bin/mocha --compilers coffee:coffee-script/register --require blanket --reporter mocha-lcov-reporter | ./node_modules/coveralls/bin/coveralls.js",
     "compile": "coffee --output lib -c src/*",
     "watch": "coffee --output lib -cw src/*"
   },
@@ -46,7 +46,7 @@
   },
   "devDependencies": {
     "coffee-script": "^1.7.0",
-    "mocha": "^2.0.1",
+    "mocha": "2.1.0",
     "mocha-lcov-reporter": "^0.0.2",
     "assert-diff": "^1.0.1",
     "blanket": "^1.1.6",

--- a/src/main.coffee
+++ b/src/main.coffee
@@ -2,7 +2,8 @@
 # A simple markdown transclusion tool
 # Author: james ramsay
 
-blanket = require 'blanket'
+
+blanket = require 'blanket' if process.env.COVERAGE
 md = require './transclude'
 dashdash = require 'dashdash'
 fs = require 'fs'

